airbyte-integrations/connectors/source-holiday-api/source_holiday_api/manifest.yaml

```diff
@@ -0,0 +1,343 @@
+version: 0.79.0
+type: DeclarativeSource
+check:
+  type: CheckStream
+  stream_names:
+    - holidays
+    - countries
+    - languages
+streams:
```

Comment:
Schemas are inlined — good, but it strikes me as odd — new versions of Builder should ref out schema definitions into a separate block. Did you use Airbyte OSS? If so, what platform version? Modern (0.57.1 and higher) should work slightly differnetly I think.